.github/workflows/release.yml

```diff
@@ -64,6 +64,8 @@ jobs:
           path: share
       - run: tar cvzf ${{ inputs.project }}.${{ inputs.target }}.tgz ${{ inputs.project }}* share
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      - run: zip -R ${{ inputs.project }}.${{ inputs.target }}.zip ${{ inputs.project }}* share
+        if: ${{ startsWith(inputs.target, 'x86_64-windows') }}
       - uses: softprops/action-gh-release@v2
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         with:
```

Comment:
You need to add the zip archive as well here. We also may need to add a `fail_on_unmatched_files: false`, see https://github.com/softprops/action-gh-release?tab=readme-ov-file#inputs.

Reply:
I agree, and I accepted both changes. Do you need me to squash my commits?

Reply:
Commits will be squashed anyway, so no need to bother.